### PR TITLE
ENH: Set the preferred Doxygen documentation style explictly.

### PR DESCRIPTION
**Pull request by jhlegarreta**
Set the preferred Doxygen documentation style explicityle following the
discussion in:
http://review.source.kitware.com/#/c/23399/

### CODE REVIEW

**Comment by jhlegarreta** [comment]
@phcerdan following the suggestion I made:
http://review.source.kitware.com/#/c/23399/

Thanks !

File: SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex [hunk 1]
```diff
@@ -271,6 +271,9 @@ \subsection{Doxygen Documentation System}
 Doxygen requires the embedding of simple comments in the code which is in turn
 extracted and formatted into documentation.
 
+Note that ITK prefers the backslash (``\\'') style versus the at-sign (``@'')
+style to wrtite the documentation commands.
```
Comment (thewtex) [inline SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex]: `wrtite the` -> `write`

Comment (jhlegarreta) [reply SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex]: Done in 4cbb0bd. Thanks @thewtex !

Comment (jhlegarreta) [reply SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex]: e481462 was rebased on master.

**Comment by jhlegarreta** [comment]
Thanks @dzenanz. @thewtex I can already merge, but it's nicer if you re-evaluate, approve and thus GitHub would be happy to merge without red warnings :smile:

**Comment by dzenanz** [comment]
I already approved, but @thewtex's change request is not "outdated".

**Comment by jhlegarreta** [comment]
Matt's change suggestion has already been addressed in e481462.

**Comment by thewtex** [comment]
Sorry for the delay in the follow-up review -- GitHub does not do a good job at sending me emails :angry:

**Pull request merged**